src/main/java/up/value/chefleaserver/dto/PopupDetailGetResponse.java

```diff
@@ -0,0 +1,35 @@
+package up.value.chefleaserver.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import up.value.chefleaserver.domain.Popup;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public record PopupDetailGetResponse(
+        String popupImage,
+        String popupName,
+        List<String> popupCategories,
+        String popupAddress,
+        @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "YY.MM.dd")
+        LocalDate popupPeriod,
+        HeadChefInfoResponse headChefInfo,
+        List<MenuResponse> menus,
+        Boolean isLiked
+) {
+    public static PopupDetailGetResponse of(Popup popup, HeadChefInfoResponse headChefInfo, Boolean isLiked){
+        return new PopupDetailGetResponse(
+                popup.getImage(),
+                popup.getName(),
+                popup.getPopupCategories().stream()
+                        .map(popupCategory -> popupCategory.getCategory().getKoreanLabel())
+                        .toList(),
+                popup.getUserRestaurant().getRestaurant().getAddress(),
+                popup.getPeriod(),
+                headChefInfo,
+                popup.getPopupmenus().stream()
+                        .map(MenuResponse::of).toList(),
```

Comment:
요기도 stream 사용하니까 개행!
```suggestion
                popup.getPopupmenus()
                        .stream()
                        .map(MenuResponse::of)
                        .toList(),
```